backend/src/main/java/com/woowacourse/levellog/prequestion/domain/PreQuestion.java

```diff
@@ -0,0 +1,64 @@
+package com.woowacourse.levellog.prequestion.domain;
+
+import com.woowacourse.levellog.common.domain.BaseEntity;
+import com.woowacourse.levellog.common.exception.InvalidFieldException;
+import com.woowacourse.levellog.levellog.domain.Levellog;
+import com.woowacourse.levellog.member.domain.Member;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.ForeignKey;
+import javax.persistence.JoinColumn;
+import javax.persistence.Lob;
+import javax.persistence.ManyToOne;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Entity
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@Getter
+public class PreQuestion extends BaseEntity {
+
+    @ManyToOne
+    @JoinColumn(nullable = false, foreignKey = @ForeignKey(name = "fk_pre_question_levellog"))
+    private Levellog levellog;
+
+    @ManyToOne
+    @JoinColumn(nullable = false, foreignKey = @ForeignKey(name = "fk_pre_question_from_member"))
+    private Member from;
+
+    @Column(nullable = false)
+    @Lob
+    private String preQuestion;
+
+    public PreQuestion(final Levellog levellog, final Member from, final String preQuestion) {
+        validateIsAuthor(levellog, from);
+        validatePreQuestion(preQuestion);
+
+        this.levellog = levellog;
+        this.from = from;
+        this.preQuestion = preQuestion;
+    }
+
+    private void validateIsAuthor(final Levellog author, final Member member) {
+        if (author.isAuthor(member)) {
+            throw new InvalidFieldException("내 레벨로그에 사전 질문을 작성할 수 없습니다.");
+        }
+    }
```

Comment:
```suggestion
    private void validateIsAuthor(final Levellog levellog, final Member member) {
        if (levellog.isAuthor(member)) {
            throw new InvalidFieldException("내 레벨로그에 사전 질문을 작성할 수 없습니다.");
        }
    }
```

타입이 levellog인데 변수명이 author인게 어색해요🥲

Reply:
이부분과 도메인 엔티티에서 잘못썼던 것 같습니다. 수정했습니다!